src/Tribe/Main.php

```diff
@@ -585,9 +586,14 @@ public function load_text_domain( $domain, $dir = false ) {
 		 */
 		$plugin_rel_path = apply_filters( 'tribe_load_text_domain', $plugin_rel_path, $domain, $locale, $dir );
 
+		// Try to load translations from wp-content/languages/plugins
 		$loaded = load_plugin_textdomain( $domain, false, $plugin_rel_path );
 
-		if ( $dir !== false && ! $loaded ) {
+		// If translation files are not found in wp-content/languages/plugins, then load from the plugin's language folder.
+		$filename = $domain . '-' . $locale . '.mo';
+		$file     = $plugin_rel_path . $filename;
+
+		if ( ! file_exists( $file ) && $dir !== false ) {
```

Comment:
A bit of explanation here:
`load_plugin_textdomain` will **always** return true unless you pass it a non-string for `$domain`.

What we really want to know is: will the "override" translation file (in `WP_LANG_DIR`) actually be loaded. If it doesn't sexists, it will not, so we use the plugin one.

Reply:
Added @lucatume and @bordoni as reviewers because of their experience with translations.